### ISSUE TITLE
We should be able to have per version documentation

### ISSUE THREAD

**Issue by Sly36** [storybookjs/storybook#12553]
**Problem**
I'm currently using Storybook v5.3 (because 6.0 has bugs with core-js), but i can't find proper documentation for this version and some of the v6 documentation and code examples do not seem to be retro compatible with 5.3.
For example in my .storybook/preview.js : 
```
export const decorators = [
    Story => (
        <div style={{ margin: '3em' }}><Story/></div>
    ),
];
```

and

```
addParameters({
    options: {
        storySort: {
            method: "alphabetical",
            order: ["Foundations", ["Colors", "Typography", "Layout"], "Components"],
        },
    },
});
```

The decorator and order do not apply to my storybook.

**Solution**
We should be able to consult the documentation for each main version of Storybook

**Comment by jonniebigodes** [comment]
@Sly36 i've picked up on your issue and wanted to give a bit of insight. With the release of 6.0 the team decided to do a massive overhaul of the documentation. I've been taking care of it. And fine tuning it where it should be fine tuned and maintained. The issue you've described it's something that i've been thinking about and see how it could be handled easily for not only you but the rest of the community. We're aware that 6.0 introduced a bit of a paradigm shift in some parts of Storybook and adjusting to that paradigm might take a bit more time. 

The documentation for 5.3 is still present in the repo if i'm not mistaken. Let me check where it is and get back to you.

**Comment by jonniebigodes** [comment]
@Sly36 the old docs are located [here](https://github.com/storybookjs/storybook/tree/release/5.3/docs/src/pages), you might have to do some digging around for what you need to lookup.

**Comment by Sly36** [comment]
Thank you for your reply ! 

For 5.3 decorators doc : https://github.com/storybookjs/storybook/blob/v5.3.18/docs/src/pages/basics/writing-stories/index.md#decorators

For 5.3 storySort doc : https://github.com/storybookjs/storybook/blob/v5.3.18/docs/src/pages/configurations/options-parameter/index.md#sorting-stories

and sortingStory order workaround : https://github.com/storybookjs/storybook/issues/548#issuecomment-660354226

**Comment by jonniebigodes** [comment]
@Sly36 going to close this issue, if you don't mind. I've recently merged a pull request(#12756) that gives some additional context on your issue. The [FAQ](https://storybook.js.org/docs/react/workflows/faq#is-it-possible-to-browse-the-documentation-for-past-versions-of-storybook) now includes a section with a table that matches current documentation with the appropriate documentation for versions 5.3 and 5.0 of Storybook.

Stay safe
